package.json

```diff
@@ -127,7 +127,6 @@
     "build": "NODE_ENV=production webpack --config web/webpack.prod.config.js --mode 'production'",
     "build-clean": "yarn run clean && yarn run build",
     "clean": "rm -rf ./web/public/build/",
-    "develop": "./node_modules/.bin/concurrently --kill-others-on-fail -n flow,webpack,translations,mypy 'yarn flow --continuous_scroll' 'yarn webpack' 'yarn cli-translations watch' 'yarn mypy --continuous_scroll'",
     "flow-check": "echo skipping flow check",
     "init-db": "scripts/db/postgres/dev/init_db.py",
     "postgres": "scripts/db/postgres/dev/connect.py",
```

Comment:
I wish I could leave comments on other unchanged lines, but `yarn translations` is also a pretty important command to have working since it's used as a part of the integration trainings to create new dimensions. The core script is included, package.json is just missing `"cli-translations": "./scripts/translations/cli_translations.py",`

Reply:
Hurm. Ok. Looks like them not being there was an oversight?

So might as well try to get them working in this PR I guess.